specification/src/main/asciidoc/platform/BrandMapping.adoc

```diff
@@ -0,0 +1,114 @@
+[cols=2, options=header]
+.Java EE to Jakarta EE Mapping
+|===
+|Java EE Technology
+|Jakarta EE Technology
+
+|Java™ Platform, Enterprise Edition 8
+|Jakarta™ EE Platform 8
+
+|Enterprise JavaBeans™ 3.2
+|Jakarta™ Enterprise Beans 3.2
+
+|JavaServer Pages™ 2.3
+|Jakarta™ Server Pages 2.3
+
+|Expression Language 3.0
+|Jakarta™ Expression Language 3.0
+
+|Common Annotations for the Java Platform 1.3
+|Jakarta™ Annotations 1.3
+
+|Java™ Servlet 4.0
+|Jakarta™ Servlet 4.0
+
+|Java™ Message Service 2.0
+|Jakarta™ Messaging 2.0
+
+|Java™ Transaction API 1.2
+|Jakarta™ Transaction 1.3
+
+|JavaMail™ API 1.6
+|Jakarta™ Mail 1.6
+
+|JavaBeans™ Activation Framework 1.1
+|Jakarta™ Activation 1.2
+
+|Java EE™ Connector Architecture 1.7
+|Jakarta™ Connectors 1.7
+
+|Web Services for Java EE 1.4
+|Jakarta™ XML Web Services 2.3
```

Comment:
Wrong mapping.  Should be "Jakarta Enterprise Web Services 1.4".

Reply:
Thank you.  Fixed in the latest version.